### PR TITLE
Changed requestPresent to accept a sequence of VRLayers

### PR DESCRIPTION
**Pull request by toji**
Also added `maxLayers` capability. As per conversations with @vvuk. Still restricts the v1 spec to only accepting a single layer, but makes future changes to allows for more complex compositing more sensible.

### CODE REVIEW

**Comment by toji** [comment]
If this change were to go into effect I would probably keep the non-array variant of the function in Chrome's experimental builds for a bit and have them print a deprecation warning to the console when used, just so any 1.0 compatible content people have built doesn't immediately fail.

**Comment by cvan** [comment]
needs rebase

File: index.bs [hunk 1]
```diff
@@ -135,7 +135,7 @@ interface VRDisplay : EventTarget {
    * Begin presenting to the VRDisplay. Must be called in response to a user gesture.
    * Repeat calls while already presenting will update the VRLayer being displayed.
```
Comment (cvan) [inline index.bs]: perhaps change to

```
will update the `VRLayer`s
```

File: index.bs [hunk 2]
```diff
@@ -135,7 +135,7 @@ interface VRDisplay : EventTarget {
...
    */
-  Promise&lt;void&gt; requestPresent(VRLayer layer);
+  Promise&lt;void&gt; requestPresent(sequence&lt;VRLayer&gt; layer);
```
Comment (cvan) [inline index.bs]: shouldn't we rename to `layers`?

Comment (cvan) [inline index.bs]: should we change [`getLayers`](https://github.com/MozVR/webvr-spec/blob/present_array/index.bs#L148) to always be required to be a sequence?

**Comment by toji** [comment]
Rebased and addressed @cvan's comments. Thanks!

**Comment by cvan** [comment]
this looks great. r+ and that's no joke :+1:

**Comment by mkeblx** [comment]
Saw this in IDL.
https://github.com/MozVR/webvr-spec/blob/master/index.idl#L44

**Comment by cvan** [comment]
oh, good catch. gotta fix that.

**Pull request merged**